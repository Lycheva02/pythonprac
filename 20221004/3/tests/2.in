(4,8,92,3,747,803,9), (5, 604, 8, 2093, 747) 
