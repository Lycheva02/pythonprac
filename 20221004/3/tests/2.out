(4, 92, 3, 803, 9)
